src/main/java/net/dv8tion/jda/api/utils/messages/MessageEditData.java

```diff
@@ -357,6 +357,22 @@ public synchronized List<FileUpload> getFiles()
                 .collect(Helpers.toUnmodifiableList());
     }
 
+    /**
+     * Returns the {@link FileUpload FileUploads} that are added indirectly to this message,
+     * such as from V2 components and embeds.
+     *
+     * @return The list of additional file uploads
+     */
+    @Nonnull
+    public synchronized List<FileUpload> getAdditionalFiles()
```

Comment:
Additional seems like confusing wording here, since its an edit request. More accurate would be `getComponentFileUploads` or similar